src/Docker.PowerShell/Docker.psm1

```diff
@@ -0,0 +1,31 @@
+#
+# Script module for module 'Docker'
+#
+# This is used to support cross platform modules and load
+# the appropriate module based on which host / PS environment
+# the module is in.
+#
+
+$PSDefaultParameterValues.Clear()
+Set-StrictMode -Version Latest
+
+$PSModule = $ExecutionContext.SessionState.Module
+$PSModuleRoot = $PSModule.ModuleBase
+
+# Import the appropriate nested binary module based on the current PowerShell version
+$binaryModuleRoot = $PSModuleRoot
+if (($PSVersionTable.Keys -contains "PSEdition") -and ($PSVersionTable.PSEdition -ne 'Desktop')) {
+    $binaryModuleRoot = Join-Path -Path $PSModuleRoot -ChildPath 'coreclr'
+}
+else
+{
+    $binaryModuleRoot = Join-Path -Path $PSModuleRoot -ChildPath 'clr' 
+}
+
+$binaryModulePath = Join-Path -Path $binaryModuleRoot -ChildPath 'Docker.PowerShell.dll'
+$binaryModule = Import-Module -Name $binaryModulePath -PassThru
+
```

Comment:
This will force the formatting to work on 6.0.  I'll file an issue against them about not auto-loading this.